multihost_test/Bugzillas/test_bz_automation.py

```diff
@@ -48,3 +48,18 @@ def test_crashes(self, multihost):
                            "grep test_user "
                            "/tmp/newroot/etc/passwd").stdout_text
         execute_cmd(multihost, "rm -fr /tmp/newroot")
+
+    def test_2093311(self, multihost, create_backup, create_localuser):
+        """
+        :title: Podman will no longer utilize UID entries
```

Comment:
The title seems confusing. I would change it for "sub*id files can also contain IDs"

Reply:
Done